Routing/Loader/DirectoryRouteLoader.php

```diff
@@ -34,6 +37,10 @@ public function __construct(RestRouteProcessor $processor)
      */
     public function load($resource, $type = null)
     {
+        if (isset($resource[0]) && '@' === $resource[0]) {
+            $resource = $this->fileLocator->locate($resource);
```

Comment:
After all I don't know ... The advantage would be to support relative paths but without using a specific type for directories I don't know how we could do that (the current path is not passed to the sub ``FileLoader``s, see [here](https://github.com/symfony/config/blob/master/Loader/FileLoader.php#L85-L87)).
So let's forget it :-/